src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/src/BlazorDebug/TerminateDebugHandler.ts

```diff
@@ -80,7 +80,7 @@ async function terminateByProcessName(
   }
 
   const devserver = processes.find(
-    (process: psList.ProcessDescriptor) => !!(process && process.cmd && process.cmd.includes(targetProcess)));
+    (process: psList.ProcessDescriptor) => !!(process && process.cmd && process.cmd.match(targetProcess)));
```

Comment:
This change is needed because above the `launchedApp` possibly became a regex right?

Reply:
Correct.